xlators/mgmt/glusterd/src/glusterd-snapshot-utils.c

```diff
@@ -1075,7 +1076,8 @@ gd_import_volume_snap_details(dict_t *dict, glusterd_volinfo_t *volinfo,
                key, volname);
         goto out;
     }
-    volinfo->is_snap_volume = (is_snap_int != 0);
+    if (is_snap_int != 0)
```

Comment:
Are we sure `GD_VOLINFO_SNAP_VOLUME` will never be set here ? otherwise we need to clear it if this is false.

Reply:
Hopefully OK as well.